test/unittests/pg.ddl/generateManyToMany.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const { join } = require('path');
+const common = require('@metarhia/common');
+const metatests = require('metatests');
+const metaschema = require('metaschema');
+const ddl = require('../../../lib/pg.ddl');
+
+const schemasDir = join(__dirname, '../..', 'fixtures/ddl-unit');
+const test = metatests.test('pg.ddl.generateManyToMany unit test');
+
+metaschema.fs.loadAndCreate(schemasDir, { common }, (err, ms) => {
+  if (err) test.bailout(err);
+
+  test.strictSame(
+    ddl.generateManyToMany(
+      'CategoryWithMany',
+      'LocalCategory1',
+      'field',
+      ms.categories,
+      'LocalCategory1'
+    ),
+    `
+CREATE TABLE "CategoryWithManyfield" (
```

Comment:
If you use string concatenation it's better to not mix it with multiline template literals.